src/helpers/getCourses.ts

```diff
@@ -0,0 +1,13 @@
+import { mockedCoursesList as data } from 'src/constants';
+
+export const getCourses = (query: string): object[] => {
+	const courses = data.map((item) => {
```

Comment:
Only `.filter` can be used here to achieve the same result

```suggestion
  return data.filter((item) => {
    return item.id.includes(query) || item.title.includes(query)
    });
```